guides/common/modules/ref_system-requirements.adoc

```diff
@@ -40,6 +40,11 @@ endif::[]
 * Administrative user (root) access
 * Full forward and reverse DNS resolution using a fully-qualified domain name
 
+{Project} only supports `UTF-8` encoding system.
```

Comment:
I'd say it's either
```suggestion
{Project} only supports the `UTF-8` encoding system.
```

But more commonly
```suggestion
{Project} only supports `UTF-8` encoding.
```

Reply:
Addressed :+1: